### PR TITLE
Task-AQL17-edit column table for show data in app

### PR DESCRIPTION
**Pull request by nguyenphuochai1998**

### CODE REVIEW

**Review by NguyenQuangKhang — APPROVED** [review]

**Review by nguyenphuochai1998 — COMMENTED** [review]
check fist it ok

**Review by nguyenphuochai1998 — COMMENTED** [review]
check fist have many space text

File: lib/widgets/basic/data_table.dart [hunk 1]
```diff
@@ -0,0 +1,165 @@
+import 'package:easysalon_mobile_ui_kit/services/layout_notifier.dart';
```
Comment (NguyenQuangKhang) [inline lib/widgets/basic/data_table.dart:1]: config Size của các Column

**Review by NguyenQuangKhang — CHANGES_REQUESTED** [review]

File: lib/widgets/basic/data_table.dart [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+import 'package:easysalon_mobile_ui_kit/services/theme_notifier.dart';
+import 'package:easysalon_mobile_ui_kit/widgets/layout/divider.dart';
+import 'package:easysalon_mobile_ui_kit/widgets/layout/divider.dart'
+    as dividerCustom;
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+class DataTableCustom extends StatelessWidget {
+  const DataTableCustom(
+      {Key? key,
```
Comment (NguyenQuangKhang) [inline lib/widgets/basic/data_table.dart:11]: thêm custom size cho các Column

**Review by NguyenQuangKhang — CHANGES_REQUESTED** [review]

**Pull request closed without merging**